Unified/checkor.py

```diff
@@ -0,0 +1,323 @@
+            print json.dumps(dbs_presence, indent=2)
+            print json.dumps(phedex_presence, indent=2)
+            ## hook for just waiting ...
+            is_closing = False
+
+        ## anything running on acdc
+        familly = getWorkflowById(url, wfi.request['PrepID'], details=True)
+        for member in familly:
+            if member['RequestName'] == wfo.name: continue
+            if member['RequestDate'] < wfi.request['RequestDate']: continue
+            if member['RequestStatus'] in ['running-opened','running-closed','assignment-approved','assigned','acquired']:
+                print wfo.name,"still has a member running",member['RequestName']
+                #print json.dumps(member,indent=2)
+                ## hook for just waiting ...
+                is_closing = False
+
+        ## for visualization later on
+        if not wfo.name in fDB.record: 
+            #print "adding",wfo.name,"to close out record"
+            fDB.record[wfo.name] = {
+            'datasets' :{},
+            'name' : wfo.name,
+            'closeOutWorkflow' : None,
+            }
+        fDB.record[wfo.name]['closeOutWorkflow'] = is_closing
+        for output in wfi.request['OutputDatasets']:
+            if not output in fDB.record[wfo.name]['datasets']: fDB.record[wfo.name]['datasets'][output] = {}
+            rec = fDB.record[wfo.name]['datasets'][output]
+            rec['percentage'] = float('%.2f'%(percent_completions[output]*100))
+            rec['duplicate'] = duplications[output]
+            rec['phedexReqs'] = float('%.2f'%any_presence[output][custodial_presences[output][0]][1]) if len(custodial_presences[output])!=0 else 'N/A'
+            rec['closeOutDataset'] = is_closing
+            rec['transPerc'] = float('%.2f'%any_presence[output][ disk_copies[output][0]][1]) if len(disk_copies[output])!=0 else 'N/A'
+            rec['correctLumis'] = (events_per_lumi[output] <= 300)
+            rec['missingSubs'] = False if len(custodial_locations[output])==0 else ','.join(custodial_locations[output])
+            rec['dbsFiles'] = dbs_presence[output]
+            rec['phedexFiles'] = phedex_presence[output]
+
+        ## and move on
+        if is_closing:
+            ## toggle status to closed-out in request manager
+            print "setting",wfo.name,"closed-out"
+            if not options.test:
+                reqMgrClient.closeOutWorkflowCascade(url, wfo.name)
+                # set it from away/assistance* to close
+                wfo.status = 'close'
+                session.commit()
+        else:
+            print wfo.name,"needs assistance"
+            ## that means there is something that needs to be done acdc, lumi invalidation, custodial, name it
+            wfo.status = 'assistance'+sub_assistance
+            if not options.test:
+                print "setting",wfo.name,"to",wfo.status
+                session.commit()
+
+    fDB.summary()
+    ## custodial requests
```

Comment:
@julianbadillo here is the custodial request for those we could not find a sunscription